scripts/services/serviceConfig.ts

```diff
@@ -0,0 +1,9 @@
+import ServiceCall from "sf-extension-utils/lib/service-call";
+export const sc = new ServiceCall({
+    baseUrl: "https://swapi.dev/api",
+    logEnabled: true,
+});
+export const scEmpty = new ServiceCall({
```

Comment:
A weird way of doing things. Why did you need something like this ?

Reply:
Because I use two kinds of http-request.

1. Getting all data.
2. Getting specific data from some link fields of my first data.

I did request to `http://swapi.dev/api/people/` and I get the data. I do clickable URL some fields. 

### For ex: 
````json
{
   "name":"Luke Skywalker",
   "height":"172",
   "mass":"77",
   "hair_color":"blond",
   "skin_color":"fair",
   "eye_color":"blue",
   "birth_year":"19BBY",
   "gender":"male",
   "homeworld":"http://swapi.dev/api/planets/1/",
   "films":[
      "http://swapi.dev/api/films/1/",
      "http://swapi.dev/api/films/2/",
      "http://swapi.dev/api/films/3/",
      "http://swapi.dev/api/films/6/"
   ],
   "species":[
      
   ],
   "vehicles":[
      "http://swapi.dev/api/vehicles/14/",
      "http://swapi.dev/api/vehicles/30/"
   ],
   "starships":[
      "http://swapi.dev/api/starships/12/",
      "http://swapi.dev/api/starships/22/"
   ],
   "created":"2014-12-09T13:50:51.644000Z",
   "edited":"2014-12-20T21:17:56.891000Z",
   "url":"http://swapi.dev/api/people/1/"
}
````

There is my first model. I do clickable to `homeworld`, `url`  fields. But I dynamically render object values I mean I don't know what the object has which keys or values. Because I created one more `ServiceCall` object and someone has no baseURL. I use the `has empty baseURL ServiceCall` object if the row is link.

I hope it's clear.

Reply:
You could have easily done that with `sc.request('/people/`, { method: "GET" ))`. Creating another service-call instance is not necessary unless you are fetching data from a different hostname.

Reply:
I am getting you now. You have an unknown url and you don't want to deal with separating it. On larger scale projects, you would need to parse the URL and use the same instance, to avoid maintenance issues. For this case, it isn't needed.